interface.js

```diff
@@ -0,0 +1,36 @@
+view.addEventListener('click', function(){
+    showPeeps();
+})
+
+search.addEventListener('click', function(){
+    let element = document.getElementById("peep").value;
+    console.log(element)
+    singlePeep(element);
+})
+
+function showPeeps(){
+    fetch('https://chitter-backend-api-v2.herokuapp.com/peeps').then(function(res){
+    return res.json()
+}).then((data) => {
+    let text = [`<ul>`]
+    let peeps = data.forEach(peep => text.push(
+      `
+      <li id="${peep.id}">${peep.body} <br> ${peep.user.handle} <br><br>
+      
+      </li>
+      `
+    ))
+    text.push(`</ul>`)
+    list.innerHTML = text.join('')
+  })
+}
+function singlePeep(id){
```

Comment:
This function doesn't look its working.

Looking at the API documentation, I think you need a user to sign up first, before being able to peep.

Reply:
I should of been more clear , it allows you to search for a single peep with a users ID which is usually a number. just realised a user who hasnt worked on it before wouldn't know how to use that function. Thanks!